test/torch/tensors/test_fv.py

```diff
@@ -676,3 +676,23 @@ def test_rns_tool_fastbconv_sk(poly_len, coeff_mod, plain_mod, input, output):
     rns_tool = RNSTool(enc_param)
     result = rns_tool.fastbconv_sk(input)
     assert result == output
+
+
+@pytest.mark.parametrize(
+    "val", [(0), (1), (-1), (100), (1000), (-1000), (-99)],
+)
+def test_fv_relin(val):
+    ctx = Context(EncryptionParams(128, CoeffModulus().create(128, [40, 40]), 64))
+    keygenerator = KeyGenerator(ctx)
+    keys = keygenerator.keygen()
+    relin_key = keygenerator.get_relin_keys()
+    encoder = IntegerEncoder(ctx)
+    encryptor = Encryptor(ctx, keys[1])  # keys[1] = public_key
+    decryptor = Decryptor(ctx, keys[0])  # keys[0] = secret_key
+    evaluator = Evaluator(ctx)
+
+    op = encryptor.encrypt(encoder.encode(val))
+    temp_prod = evaluator.mul(op, op)
+    relin_prod = evaluator.relin(temp_prod, relin_key)
+    assert len(temp_prod.data) - 1 == len(relin_prod.data)
```

Comment:
Can we add more tests, ones that do multiple mul and relin, and others that check the failure for len 2 and >3 ciphertexts?